### ISSUE TITLE
Docker isn't running HNN. Help please

### ISSUE THREAD

**Issue by raulrpr1** [jonescompneurolab/hnn#241]
I'm trying to install HNN following this tutorial https://jonescompneurolab.github.io/hnn/installer/windows/docker-toolbox.html
but when I try to run it ( ./hnn_docker.sh -u start) , it displays me an error :

$ ./hnn_docker.sh -u start
======================================
Upgrade HNN image requested

Starting HNN container requested

Performing pre-checks before starting HNN
--------------------------------------
Checking OS version... windows
Checking if docker is installed... done
Checking if Docker (Toolbox) is working... done
Checking Docker container type... linux
Downloading new HNN image from Docker Hub... done
Looking for existing containers... found, running up to date image
Checking if VcXsrv is running... no
Checking for vcxsrv... found
Starting VcXsrv... done
Checking for xauth... found
Checking for X11 authentication keys... done
Updating xauth keys for use with docker... done
Checking for SSH authentication files... done
Checking permissions within /c/Users/raulr/hnn_out... done

Starting HNN
--------------------------------------
Checking for running HNN container... found
Checking permissions of /c/Users/raulr/hnn_out in container... done
Checking if X server is reachable from container... *failed*
Stopping VcXsrv... done

======================================
Error: Please see /c/Users/raulr/hnn/hnn_docker.log for more details

When I check the log (using /c/Users/raulr/hnn/hnn_docker.log) , this is the answer:
$ /c/Users/raulr/hnn/hnn_docker.log
/c/Users/raulr/hnn/hnn_docker.log: line 2: Performing: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 3: --------------------------------------: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 5: Checking: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 6: windows: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 8: Checking: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 10: __init__.py: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 11: __init__.py: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 12: __init__.py: command not found
/c/Users/raulr/hnn/hnn_docker.log: line 13: syntax error near unexpected token `done'
/c/Users/raulr/hnn/hnn_docker.log: line 13: `done'

When I only type hnn_docker.log, this is the answer:
bash: hnn_docker.log: command not found

**Comment by blakecaldwell** [comment]
@raulrpr1 thanks for posting this. We will be updating the install method for Windows 10 Home, which has been problematic using Docker. I will post a comment to this issue to let you know when the instructions have been added (within a few days).

**Comment by blakecaldwell** [comment]
@raulrpr1 I recently merged #218 which may fix the problem you were having. If you could try again with the updated code, please let me know the result.

After changing to the installation directory:
```
git pull origin master
./hnn_docker.sh -u start
```

BTW you can view the log file with a text editor (e.g. notepad) in the `hnn` directory. If you are still running into issues, please attach the log file to this issue by dragging and dropping into a comment. Thanks.

**Comment by blakecaldwell** [comment]
This problem may be fixed in recently merged commits, but need a `hnn_docker.log` file to confirm the reason behind
```
Checking if X server is reachable from container... failed
```

Closing for now. This will be a legacy installation method in the future, but in case others run into this, I'm changing the title to reflect the error.
